homeassistant/components/bayesian/strings.json

```
@@ -14,5 +14,195 @@
+        },
+        "data_description": {
+          "probability_threshold": "[%key:component::bayesian::config::step::user::data_description::probability_threshold%]",
+          "prior": "[%key:component::bayesian::config::step::user::data_description::prior%]"
+        }
+      }
+    }
+  },
+  "config": {
+    "error": {
+      "extreme_prior_error": "'Prior' set to 0% means that it is impossible for the sensor to show 'on' and 100% means it will never show 'off', use a close number like 0.1% or 99.9% instead",
+      "extreme_threshold_error": "'Probability threshold' set to 0% means that the sensor will always be 'on' and 100% mean it will always be 'off', use a close number like 0.1% or 99.9% instead",
+      "equal_probabilities": "If 'Probability given true' and 'Probability given false' are equal, this observation can have no effect, and is therefore redundant",
+      "extreme_prob_given_error": "If either 'Probability given false' or 'Probability given true' is 0 or 100 this will create certainties that override all other observations, use numbers close to 0 or 100 instead"
+    },
+    "step": {
+      "user": {
+        "title": "Add a Bayesian sensor",
+        "description": "Create a binary sensor which observes the state of multiple sensors to estimate whether an event is occurring, or if something is true. See [the documentation](https://www.home-assistant.io/integrations/bayesian/) for more details.",
+        "data": {
+          "probability_threshold": "Probability threshold",
+          "prior": "Prior",
+          "device_class": "Device class",
+          "name": "[%key:common::config_flow::data::name%]"
+        },
+        "data_description": {
+          "probability_threshold": "The probability above which the sensor will show as 'on'. 50% should produce the most accurate result. Use numbers greater than 50% if avoiding false positives is important, or vice-versa.",
+          "prior": "The baseline probabilty the sensor should be 'on', this is usually the percentage of time it is true. For example, for a sensor 'Everyone sleeping' it might be 8 hours a day, 33%.",
+          "device_class": "Choose the device class you would like the sensor to show as."
+        }
+      }
+    }
+  },
+  "config_subentries": {
+    "observation": {
+      "step": {
+        "user": {
+          "title": "Add an observation",
+          "description": "'Observations' are the sensor or template values that are monitored and then combined in order to inform the Bayesian sensor's final probability. Each observation will update the probability of the Bayesian sensor if it is detected, or if it is not detected. If the state of the entity becomes `unavailable` or `unknown` it will be ignored. If more than one state or more than one numeric range is configured for the same entity then inverse detections will be ignored.",
+          "menu_options": {
+            "state": "Add on observation for a sensor's state",
```

This looks like a typo:

```suggestion
            "state": "Add an observation for a sensor's state",
```